Dockerfile.centos7

```diff
@@ -21,4 +21,5 @@ VOLUME /config /secrets /cache
 
 WORKDIR /home/tester
 ENTRYPOINT ["/usr/local/bin/dumb-init", "--"]
+CMD ["cp", "/test/standard-inventory-qcow2-python3", "/usr/share/ansible/inventory/standard-inventory-qcow2"]
```

Comment:
Rather than having a copy of the script in our repo that we have to maintain, I would rather just download the script from the source when we build the container e.g. `curl -s -o /usr/share/ansible/inventory/standard-inventory-qcow2 https://pagure.io/standard-test-roles/raw/ce0ce5eb049c886a0cb4ecbd3f4c9b409693ba4a/f/inventory/standard-inventory-qcow2`
One wrinkle is that you cannot apparently download from a git tag from pagure like you can from github, so that's why the link above uses the full git commit hash `ce0ce5eb049c886a0cb4ecbd3f4c9b409693ba4a` instead of a tag.  If you go to the Releases page - https://pagure.io/standard-test-roles/releases - you can see that the latest release is `4.10`  - if you go to the files, you can see that it corresponds to the commit hash.

Reply:
It make sense. I updated it the latest commit.